src/flinkSql/languageClient.ts

```diff
@@ -16,7 +16,7 @@ import { getFlinkSQLLanguageServerOutputChannel } from "./logging";
 import { WebsocketTransport } from "./websocketTransport";
 
 const logger = new Logger("flinkSql.languageClient.Client");
-
+const FLINK_DIAGNOSTIC_COLLECTION_NAME = "confluent.flinkSql";
```

Comment:
nit: Keep the empty line

```suggestion
const FLINK_DIAGNOSTIC_COLLECTION_NAME = "confluent.flinkSql";

```